package.json

```diff
@@ -19,10 +19,13 @@
         "@types/node": "^12.12.7",
         "autoprefixer": "9.4.3",
         "minimist": "1.2.3",
-        "postcss": "7.0.7",
+        "postcss": "^7.0.7",
         "rtlcss": "^2.4.0",
         "typescript": "^3.7.2"
     },
+    "peerDependencies": {
+        "postcss": "^7.0.7"
+    },
     "devDependencies": {
         "@bazel/typescript": "^1.0.1"
     },
```

Comment:
bazelWorkspaces on the next line is also vestigial in rules_nodejs 2.0

Reply:
Removed